Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -28,9 +30,21 @@ YUI.add('ez-usermodel-tests', function (Y) {
                     "login": "admin",
                     "email": "dp@ez.no",
                     "enabled": "true",
-                    "name": "Administrator User"
-                }
+                    "name": "Administrator User",
+                    "Version":  {
+                        "Fields": {
+                            "field": []
+                        }
+                    }
+                },
+                "_contentType": this.contentType
             };
+            Y.Mock.expect(this.contentType, {
+                method: this.getFieldDefinitionIdentifiers,
```

Comment:
I don't really see the point in putting the method name in a property of the test object. You can directly put the literal string here.